src/main/java/com/dougnoel/sentinel/strings/SentinelStringUtils.java

```diff
@@ -132,4 +132,62 @@ public static String replaceVariable(String text) {
 		return text;
 	}
 
+	/**
+	 * Returns a string with any strings with pattern "\$(int|str) \{.*?\}" replaced with values
+	 * stored configuration. For example, given the string
+	 {
+	 "id": $int {id},
+	 "name": "puppy",
+	 "category": {
+	 "id": 1,
+	 "name":  $str {category_name}
+	 			}
+	 }
+	 * This method would return: {
+	 * 	 "id": 10,
+	 * 	 "name": "puppy",
+	 * 	 "category": {
+	 * 	 "id": 1,
+	 * 	 "name": "Dog"
+	 *    }}.
+	 * If there are no values to replace, this method returns the string intact.
+	 * @param text String the text to search for variable replacement
+	 * @return String the string with variables replaced as applicable
+	 */
+	public static String replaceStoredVariables(String text) {
```

Comment:
If I'm honest I don't like the design of the replacement syntax. If you need to surround something with quotes, why not just have the quotes in the template? For example, if you need the result to look like `"name": "Dog"`, why not make the input `"name": "{category_name}"`. It seems to me that's the only distinction made between `int` and `str` here, but I think it makes more sense to not have any such distinction. If there is a question of whether `{variable_name}` is unique enough, then we can change the syntax to something more unique without adding type info to the front.